assets/scss/_styles_project.scss

```diff
@@ -37,3 +37,57 @@
     &.navbar-bg-onscroll { background: $dark !important }
 }
 
+/* Click-to-copy button styles */
+
+.highlight {
+   
+   .click-to-copy {
+       display: block;
+       text-align: right;
+       margin-right: -8px;
+       margin-top: -10px;
```

Comment:
When the code has horizontal scroll the button scrolls with it and can go off the left edge.

To keep it pinned to the top rigth we could set `position: relative` on the container, `position: absolute` on the button, and change these margins to positions `right: 8px; top: 10px;` instead

Reply:
Great catch! I didn't test what scrolling does to the button 🤦

I applied your suggestions with small adjustment to the top value and fixed high for the button container to prevent the code from overlapping the button.